wasm/test_page/start_server.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
-
+echo "Start: Copying artifacts in local folder------"
 cp ../../build-wasm/wasm/bergamot-translator-worker.data .
 cp ../../build-wasm/wasm/bergamot-translator-worker.js .
 cp ../../build-wasm/wasm/bergamot-translator-worker.wasm .
 cp ../../build-wasm/wasm/bergamot-translator-worker.worker.js .
+echo "Done----"
+
+echo "Start: Enabling wormhole via APIs that compile and instantiate wasm module-------"
```

Comment:
At this point, the artifacts should already be patched, so we should not patch them again (assuming that the user has followed the build steps outlined above)

Reply:
Agreed.